packages/core/src/generatePolicy.js

```diff
@@ -160,6 +213,7 @@ function createModuleInspector(opts = {}) {
     if (includeDebugInfo) {
       const moduleDebug = debugInfo[moduleRecord.specifier]
       moduleDebug.sesCompat = {
+        // XXX: I don't think this is needed, since it appears we overwrite all properties
```

Comment:
(nit) Let's use one of the TODO: or FIXME: if this comment is here to stay and not just meant for the review. 

I don' know what's the full shape of sesCompat.

Reply:
hmm, I'll apply this suggestion to #732